### PR TITLE
Registering URL Schemes for Signed macOS Apps Is Not Possible

### PR DESCRIPTION
**Pull request by BeckerWdf**
Contributes to https://github.com/eclipse-platform/eclipse.platform.ui/issues/1901

### CODE REVIEW

File: eclipse.platform.common/bundles/org.eclipse.platform.doc.user/reference/ref-linkhandlersprefs.htm [hunk 1]
```diff
@@ -70,6 +70,11 @@ <h3>Note</h3>
     <li>For the "macOS" and "Linux" operating system: If an URL scheme is handled by another application (Eclipse-based
     or not), it <strong>cannot</strong> be enabled directly. Handling of this URL scheme must first be disabled in the
     other application, before it can be enabled in the current application.</li>
+    <li>For the "macOS" operating system: If the current application is a signed application it's not possible to 
+    register / unregister URL schemes. Registering / unregistering would modify the application's info.plist file.
+    This would break the applications signature and lead to the situation that "macOS Gatekeeper" would not longer allow
+    to run the application. URL schemes must be added to the info.plist file already at build-time of the application
+    becore it's singed. For unsigned applications this restriction does not apply.</li>
```
Comment (merks) [inline eclipse.platform.common/bundles/org.eclipse.platform.doc.user/reference/ref-linkhandlersprefs.htm:77]: singed -> signed

Comment (iloveeclipse) [reply eclipse.platform.common/bundles/org.eclipse.platform.doc.user/reference/ref-linkhandlersprefs.htm:77]: Sure :-) ?

Comment (merks) [reply eclipse.platform.common/bundles/org.eclipse.platform.doc.user/reference/ref-linkhandlersprefs.htm:77]: Either that or singed -> sang but there are no songs involved here.  😛

File: eclipse.platform.common/bundles/org.eclipse.platform.doc.user/reference/ref-linkhandlersprefs.htm [hunk 2]
```diff
@@ -70,6 +70,11 @@ <h3>Note</h3>
     <li>For the "macOS" and "Linux" operating system: If an URL scheme is handled by another application (Eclipse-based
     or not), it <strong>cannot</strong> be enabled directly. Handling of this URL scheme must first be disabled in the
     other application, before it can be enabled in the current application.</li>
+    <li>For the "macOS" operating system: If the current application is a signed application it's not possible to 
+    register / unregister URL schemes. Registering / unregistering would modify the application's info.plist file.
+    This would break the applications signature and lead to the situation that "macOS Gatekeeper" would not longer allow
```
Comment (tomaswolf) [inline eclipse.platform.common/bundles/org.eclipse.platform.doc.user/reference/ref-linkhandlersprefs.htm:75]: 1. "no longer"
2. AFAIK "allow + infinitive" needs an object. Use a gerund: "would no longer allow running the application", or make the application the object: "would no longer allow the application to run".

**Pull request merged**